WinImageBuilder.psm1

```diff
@@ -635,8 +635,13 @@ function Resize-VHDImage {
         Write-Host "New partition size: $newSizeGB GB"
 
         if ($NewSize -gt $MinSize) {
+            $global:i = 0

```

Comment:
beside $global, any other way to do "closures" in the Execute-Retry script block?